units/crons.go

```diff
@@ -1210,3 +1211,23 @@ func populateQueueGroup(ctx context.Context, env evergreen.Environment, queueGro
 
 	return errors.Wrapf(amboy.EnqueueManyUniqueJobs(ctx, queueGroup, jobs), "populating '%s' queue", queueGroupName)
 }
+
+func logGithubAPILimit(env evergreen.Environment) amboy.QueueOperation {
+	return func(ctx context.Context, queue amboy.Queue) error {
+		limit, err := thirdparty.CheckGithubAPILimit(ctx)
+		if err != nil {
+			return errors.Wrap(err, "checking GitHub API rate limit")
+		}
+
+		grip.Info(message.Fields{
+			"message":           "Evergreen GitHub API rate limit",
```

Comment:
nit: can we keep this "GitHub API Rate limit" so any splunk things won't break? (Also since we always intended for this to just be evergreen i think it speaks for itself)